app/dts/bindings/animations/zmk,animation-ripple.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2020, The ZMK Contributors
```

Comment:
Just a general comment- copyright year needs to be updated for most of these files